src/external/views/nunjucks/returns/meter-reset.njk

```diff
@@ -1,10 +1,10 @@
 {% extends "./nunjucks/layout.njk" %}
 {% from "forms/index.njk" import formRender, formErrorSummary %}
-{% from "nunjucks/returns/macros/return-header.njk" import returnHeader %}
+{% from "returns/return-header.njk" import returnHeader %}
 
 {% block content %}
   {{ formErrorSummary(form) }}
-  {{ returnHeader(data, documentHeader, showMeta, isAdmin) }}
+  {{ returnHeader(data, documentHeader, showMeta) }}
   {{ formRender(form) }}
 {% endblock %}
 
```

Comment:
I think we should probably still use `var` instead of `const` in front-end JS.

Is there any way this functionality could be generalised and placed in the main application JS?  E.g. in the same way the clickable tables was implemented with the data attribute.

This template would then no longer be necessary.